content/applications/hr/recruitment/source_analysis.rst

```diff
@@ -0,0 +1,86 @@
+=========================
+Source analysis reporting
+=========================
+
+Reporting is a critical tool used by recruiting departments to gain insights into the entire
+recruitment process.
+
+Determining where applicants come from can provide information about which sources have the best
+results. This information is determined by the *Source Analysis* report. With this data, recruitment
+teams can better pivot their recruiting strategies to gain better applicants, in both quantity and
+quality.
+
+Open report
+===========
+
+To access the *Source Analysis* report, navigate to :menuselection:`Recruitment app --> Reporting
+--> Source Analysis`.
+
+This presents the data for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`, in a default
+:icon:`fa-area-chart` :guilabel:`(Graph)` view, showing the amount of applicants by
+:guilabel:`Source`, and further separated by stage (:guilabel:`In Progress` and :guilabel:`Hired`)
+
+Hover the cursor over any column to view the specific numbers fort that column.
+
+.. image:: source_analysis/source-analysis.png
+   :align: center
+   :alt: The default bar chart of the source analysis information.
```

Comment:
I'd like to see this image retaken with more zoom, since the text is currently illegible 🙏